changelog.md

```diff
@@ -7,6 +7,9 @@
 -   [t]—test suite improvement
 -   [d]—docs improvement
 
+## 2.7.2 (March 24, 2023)
```

Comment:
This would add a feature and thus turn it into 2.8.0
I'd change the 2.7.1 below to 2.8.0 and move the line to down there.
It's XX because XX is the day of the release, not the day of the merge or commit